dlt/helpers/airflow_helper.py

```diff
@@ -86,7 +86,7 @@ def __init__(
         This happens **after all dlt loading jobs executed**. See more here: https://dlthub.com/docs/running-in-production/running#failed-jobs
 
         `abort_task_if_any_job_failed` will abort the other dlt loading jobs and fail the Airflow task in any of the jobs failed. This may put your warehouse in
-        inconsistent state so the option is disabled by default.
+        inconsistent state. See https://dlthub.com/docs/running-in-production/running#handle-exceptions-failed-jobs-and-retry-the-pipeline.
```

Comment:
Setting the destination to an inconsistent state sounds scary. However, it's a necessary inconvenience, correct?